NEWS

```diff
@@ -1,5 +1,5 @@
 #
-## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001-2021. ALL RIGHTS RESERVED.
+## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001-2024. ALL RIGHTS RESERVED.
```

Comment:
```suggestion
## Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2001. ALL RIGHTS RESERVED.
```

Reply:
Why do you suggest to revert it?

Reply:
It should be only the year of creation according to the guidance.